jest/helpers.ts

```diff
@@ -11,39 +11,43 @@ import {Writable} from 'readable-stream';
 
 const CLI_PATH = path.resolve(__dirname, '../packages/cli/build/bin.js');
 
-type RunOptions = {
+type RunCliOptions = {
   nodeOptions?: string;
   nodePath?: string;
   timeout?: number; // kill the process after X milliseconds
   expectedFailure?: boolean;
 };
 
-type CLIArgs = {
-  dir: string;
-  args: string[] | undefined;
-  options: RunOptions;
-};
-
-export function run(
+/**
+ * Helper function to run CLI command in a given folder
+ */
+export function runCli(
   dir: string,
   args?: string[],
-  options: RunOptions = {
+  options: RunCliOptions = {
```

Comment:
can we rename to `RunCLIOptions`?

Reply:
No strong feelings - I wanted to keep it aligned with `runCli` command. What you think? Do we have any kind of naming policy in place to help here?

Reply:
Guess it can be just `RunOptions` as before.